### PR TITLE
translation(pt-br): Translate half of the features section

### PR DESCRIPTION
**Pull request by felipesuri**
Continuing Portuguese Brazil translation of the docs. (#1987)

- [x] Translate half of the features section.

## Translated pages:

- [x] 1.rendering-modes
- [x] 2.deployment-targets
- [x] 3.file-system-routing
- [x] 4.data-fetching

### CODE REVIEW

File: content/pt-br/docs/3.features/1.rendering-modes.md [hunk 1]
```diff
@@ -1,30 +1,31 @@
...
 ```js{}[nuxt.config.js]
 export default {
-  ssr: true // default value
+  ssr: true // Valor padrão
 }
 ```
 
 ::alert{type="info"}
-You do not need to add `ssr: true` to your nuxt config in order to enable server-side-rendering as it is enabled by default.
+Você não precisa adicionar `srr: true` em sua configuração do Nuxt para habilitar a renderização do lado do servidor, pois ela é habilitada por padrão.
```
Comment (smarroufin) [inline content/pt-br/docs/3.features/1.rendering-modes.md:23]:
```suggestion
Você não precisa adicionar `ssr: true` em sua configuração do Nuxt para habilitar a renderização do lado do servidor, pois ela é habilitada por padrão.
```

File: content/pt-br/docs/3.features/4.data-fetching.md [hunk 2]
```diff
@@ -124,55 +124,55 @@ In addition to fetch being called by Nuxt, you can manually call fetch in your c
...
+    // Chamando também em mudanças de query
   }
 }
 ```
 
-### Caching
+### Cache
 
-You can use `keep-alive` directive in `<nuxt/>` and `<nuxt-child/>` component to save `fetch` calls on pages you already visited:
+Você pode utilizar a diretiva `keep-alive` nos componentes `<nuxt />` e `</nuxt-child>` para salvar as chamadas ao `fetch` em páginas que você visitou.
```
Comment (smarroufin) [inline content/pt-br/docs/3.features/4.data-fetching.md:147]:
```suggestion
Você pode utilizar a diretiva `keep-alive` nos componentes `<nuxt />` e `<nuxt-child />` para salvar as chamadas ao `fetch` em páginas que você visitou.
```

**Review by smarroufin — CHANGES_REQUESTED** [review]
Nice work, thanks! 💚 

LGTM except those 2 typos.
Also waiting for @ChristopheCVB approval.

File: content/pt-br/docs/2.concepts/1.views.md [hunk 3]
```diff
@@ -45,7 +45,7 @@ Cada Página é um componente Vue, mas o Nuxt adiciona atributos e funções esp
 Há muitas propriedades em uma página, tais como a propriedade head no exemplo acima.
 
 ::alert{type="next"}
-Consulte [Estrutura de Diretório](/docs/directory-structure/nuxt) para aprender mais sobre todas as propriedades que podem ser utilizadas na sua página.
+Consulte [Estrutura de Diretório](/docs/directory-structure/pages) para aprender mais sobre todas as propriedades que podem ser utilizadas na sua página.
```
Comment (ChristopheCVB) [inline content/pt-br/docs/2.concepts/1.views.md:48]: You sure this is the right link?

File: content/pt-br/docs/3.features/2.deployment-targets.md [hunk 4]
```diff
@@ -1,47 +1,48 @@
...
 ```
 
-**Running nuxt dev with the static target will improve the developer experience:**
+**Rodar `nuxt dev` com o `target: static` melhorará a experiência do desenvolvedor:**
 
-- Remove `req` & `res` from `context`
-- Fallback to client-side rendering on 404, errors and redirects [see SPA fallback](/docs/concepts/static-site-generation#spa-fallback)
-- `$route.query` will always be equal to `{}` on server-side rendering
-- `process.static` is true
+- Remove `req` e `red` do `context`
```
Comment (ChristopheCVB) [inline content/pt-br/docs/3.features/2.deployment-targets.md:26]:
```suggestion
- Remove `req` e `res` do `context`
```

**Review by ChristopheCVB — CHANGES_REQUESTED** [review]
Just some typos to fix

**Comment by felipesuri** [comment]
Done @ChristopheCVB and @smarroufin.

**Review by ChristopheCVB — APPROVED** [review]
👍 Terrific work, again!

**Review by smarroufin — APPROVED** [review]
Thanks again! 💚

**Pull request merged**